### PR TITLE
Add examples for AssistantV2

### PR DESCRIPTION
**Pull request by AnthonyOliveri**

### CODE REVIEW

File: AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import AssistantV2
+
+let assistant = setupAssistantV2()
+var assistantID = WatsonCredentials.AssistantV2ID
+
+// Setup
+
+var sessionID = ""
+
+assistant.createSession(assistantID: assistantID) {
```
Comment (mkistler) [inline AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift:17]: Could we move the setup and cleanup code to the Helpers file, just to minimize the clutter on this page?

**Review by mkistler — CHANGES_REQUESTED** [review]
Looks good, but I would like the setup and cleanup code to move to the helpers file.

File: AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift (hunk 1 above)
Comment (AnthonyOliveri) [reply AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift:17]: I thought we should do that too, but I did it this way to match how it's done in other playgrounds (like AssistantV1 Synonyms). If you prefer changing all playgrounds to move setup/teardown code to the Helpers.swift, I'm onboard with that.

Comment (mkistler) [reply AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift:17]: Yes, let's put the setup/cleanup code in helpers in all the playgrounds. I think this is something I started doing later and didn't go back to fix in the earlier ones.

Comment (AnthonyOliveri) [reply AssistantV2.playground/Pages/Message.xcplaygroundpage/Contents.swift:17]: Ok. I'll merge this PR as-is, and start another PR to move the setup/teardown code for all of the services.

**Pull request merged**